pkg/shp/flags/buildrun.go

```diff
@@ -0,0 +1,49 @@
+package flags
+
+import (
+	buildv1alpha1 "github.com/shipwright-io/build/pkg/apis/build/v1alpha1"
+	"github.com/spf13/pflag"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// BuildRunSpecFlags BuildRun's spec represtantation as command-line flags.
+func BuildRunSpecFlags(flags *pflag.FlagSet) *buildv1alpha1.BuildRunSpec {
```

Comment:
Style nit - make the function name and godoc more descriptive.

```suggestion
// BuildRunSpecFromFlags creates a BuildRun spec from command-line flags.
func BuildRunSpecFromFlags(flags *pflag.FlagSet) *buildv1alpha1.BuildRunSpec {
```